src/controller.cpp

```diff
@@ -6,6 +6,14 @@
 #include "view.h"
```

Comment:
where is DB creation code? (table, fields crtration?)

Reply:
file - "db_connection.py"
line #30 
"create_table = winner (
	session_id datetime,
	number int, 
);"
I am so sorry for only 2 fields